testdata/go/v2/memcached-operator/PROJECT

```diff
@@ -2,7 +2,13 @@ domain: example.com
 layout:
 - go.kubebuilder.io/v2
 plugins:
-  manifests.sdk.operatorframework.io/v2: {}
+  envtest.sdk.operatorframework.io/v1: {}
+  manifests.sdk.operatorframework.io/v2:
+    resources:
+    - domain: example.com
+      group: cache
+      kind: Memcached
+      version: v1alpha1
```

Comment:
@Adirio @estroz imo it is the only blocker. Can we keep the same behaviour as before? I mean, not tracking the resources? WDYT? Or do you prefer starting to track now @estroz? 

Otherwise, it has my OK

Reply:
IMO this doesn't affect the user and will start tracking the resources. This will probably be needed in a future and if we start tracking them now, less resources will need to be added manually/by tooling when we require them.

Reply:
Do, we really need to track the resources for this plugin? Note that is valid for the full project. It is only called when we create any api because we want to add this value in the kustomize file to let the generate bundle know what APIs should be used and added in the CSV. 

So, in the future, we could indeed make the plugin smarter and ensure that when it is called it will always check all project resources and update the kustomize file. Ditto so, I do not think that this info is required to be tracked for this plugin and then, I would prefer we not add a new behaviour unless that is required. 

Otherwise, it indeed can let us need to ask for users to manually delete or change that in the future for example.

Reply:
The case where some APIs may be using this plugn and some no is the following:
```
operator-sdk init
operator-sdk create api --kind KindOne ...
kubebuilder create api --kind KindTwo ...
```
`KindOne` will have used `manifests` plugin but `KindTwo` won't, as this plugin is not know by `kubebuilder`.

Reply:
Using your example, if we change the plugin or sdk to before generating the bundle, for example, check all resources in the PROJECT file and add them to the kustomize then that is solved right? 

So, I'd prefer not to add this now and before that to see if we could improve the plugin or sdk helper for they are able to work with KB scaffolds first. Then, after we check all options we might decide to track or not this info. Is that make sense? Could I clarify my motivations/ideas here properly?   

However, let's what @estroz and/or @jmrodri thinks about that as well.

Reply:
I will track an issue with my ideas/motivations and link here to clarify. IMO I think we could first see how/if we can address these requirements before, take this decision.

Reply:
In order to do what you propose if we dont have this tracked we would have to "guess" if they were already added. If we add this we would "know" if they are added. IMO knowledge > guesses.

Reply:
it is true. Let's do one thing, since I have no big objections as well I will close this one and then we can just ask for both if they are ok with OK.

Reply:
This seems more like a workaround to the SDK’s Go plugin bundle overriding the kubebuilder plugin. These resources aren’t used anywhere right now either. I vote to remove them. Also the envtest plugin isn’t really a plugin per se, so that should not write a plugin key.